assets/app/views/project.html

```diff
@@ -75,7 +75,8 @@ <h2 class="service" ng-if="!displayRouteByService[serviceId]">
                 <span ng-if="service.spec.portalIP != 'None'">
                   routing traffic on {{service.spec.portalIP}}
                 </span>
-                <span ng-if="numPorts">&#8211;
+                <span ng-if="numPorts">
+                  <br>
```

Comment:
Suggest removing the `br` and just change this to a `div`, which has a natural line break.